libraries/AP_Notify/AP_Notify.h

```diff
@@ -138,10 +140,18 @@ class AP_Notify
     AP_Int8 _rgb_led_override;
     AP_Int8 _buzzer_enable;
     AP_Int8 _display_type;
+    AP_Int8 _oled_theme;
```

Comment:
To be consistent with the parameter name change suggested above, maybe we should rename this member to _oreo_theme as well? I'm just thing we can be consistent in using "oreo" instead of using both "oreo" and "oled".